contracts/Core.sol

```diff
@@ -99,4 +106,12 @@ contract Core is CoreInterface {
 	{
 		return coreOpenSTRemote;
 	}
+
+	function safeUnlockTime()
+		external
+		view
+		returns (uint256 unlockTime)
+	{
+		return blocksToWaitMedium + block.number;
```

Comment:
to be sure: it is the latest seen `_blockHeight` submitted during `commitStateRoot()`, you might have been holding off on MOSA-62 to get the variable.

`return blocksToWaitMedium + latestStateRootBlockHeight`